assets/css/style.css

```diff
@@ -0,0 +1,66 @@
+@charset "UTF-8";
```

Comment:
Can you please add a screenshot of how the page looks? That would really make reviewing easy.